### PR TITLE
Make cargo test work

### PR DESCRIPTION
**Pull request by michaelwu**
Fixes https://github.com/servo/rust-mozjs/issues/161

### CODE REVIEW

File: src/rust.rs [hunk 1]
```diff
@@ -728,9 +728,11 @@ pub unsafe extern fn reportError(_cx: *mut JSContext, msg: *const c_char, report
 #[cfg(test)]
 pub mod test {
     use super::Runtime;
+    use jsapi::JS_Init;
 
     #[test]
     pub fn dummy() {
+        unsafe { JS_Init(); }
```
Comment (Ms2ger) [inline src/rust.rs]: Check the result, please.

**Comment by Ms2ger** [comment]
I don't understand the linkhack change, but then again, I don't know if anybody does. Does Servo compile with this change?

File: src/rust.rs (hunk 1 above)
Comment (michaelwu) [inline src/rust.rs]: We don't even do that in servo.

**Comment by michaelwu** [comment]
Yeah, I checked on Linux.

**Comment by michaelwu** [comment]
I mostly understand what's going on with the linkhack now. I just don't know how it worked before.

**Comment by michaelwu** [comment]
Commit updated to use assert_eq, same as in servo.

**Pull request merged**